test/fixtures/runtimeGasUsed.js

```diff
@@ -1,196 +1,201 @@
 'use strict';
 module.exports = [
-  42532,
-  42566,
-  42596,
-  42630,
-  42662,
-  42694,
-  42662,
-  43447,
-  43479,
-  43148,
-  43358,
-  43568,
-  43778,
-  49658,
-  42854,
-  43818,
-  42852,
-  42948,
-  42980,
-  43012,
-  42364,
-  43076,
-  43108,
-  43140,
-  44476,
-  43140,
-  43236,
-  43204,
-  43236,
-  43712,
-  44332,
-  58383,
-  58415,
-  58447,
-  58479,
-  58511,
-  58543,
-  58575,
-  58607,
-  58575,
-  58607,
-  58703,
-  58671,
-  58767,
-  58799,
-  58831,
-  45298,
-  46264,
-  47166,
-  48132,
-  49163,
-  50129,
-  51096,
-  52062,
-  53029,
-  53995,
-  54898,
-  55865,
-  56896,
-  57863,
-  58766,
-  59797,
-  43566,
-  42053,
-  42117,
-  42149,
-  42106,
-  41455,
-  43119,
-  41679,
-  42554,
-  42522,
-  41775,
-  41807,
-  42967,
-  43444,
-  60598,
-  44194,
-  43162,
-  56816,
-  42397,
-  42179,
-  49888,
-  43921,
-  43921,
-  43921,
-  43857,
-  43921,
-  43921,
-  43921,
-  45333,
-  45333,
-  45333,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  43921,
-  45333,
-  45337,
-  43921,
-  43797,
-  43921,
-  43921,
-  43925,
-  43921,
-  45425,
-  43921,
-  43949,
-  44020,
-  43459,
-  43432,
-  42578,
-  46735,
-  42482,
-  45650,
-  43320,
-  44758,
-  43288,
-  45303,
-  48730,
-  45749,
-  46594,
-  43467,
-  43987,
-  43086,
-  45528,
-  46558,
-  47524,
-  48554,
-  49649,
-  44811,
-  44381,
-  45035,
-  44590,
-  44063,
-  43499,
-  43809,
-  42356,
-  47334,
-  50672,
-  65348,
-  62849,
-  63721,
-  112510,
-  149250,
-  69726,
-  73273,
-  53805,
-  45814,
-  51598,
-  50696,
-  49966,
-  48221,
-  48871,
-  50468,
-  55360,
-  71277,
-  50498,
-  52503,
-  68307,
-  50588,
-  52946,
-  68858,
-  50614,
-  50983,
-  66856,
-  51861,
-  51568,
-  47387,
-  46549,
-  50248,
-  45271,
-  39755,
-  44654,
-  44526,
-  52864,
-  64225,
-  62240,
-  68059,
-  71245,
-  795382,
-  107298,
-  756224,
-  243265,
-  331690,
-  765537,
-  47813,
+  90953,
```

Comment:
how does this one get created?

Reply:
This entire file is generated during tests.

Reply:
should we exclude it from git then?

Reply:
Maybe 🤔 
I did not take the time to fully understand what it does 😆 , but I assume it is committed for a reason.